### ISSUE TITLE
Safari issue

### ISSUE THREAD

**Issue by nneuh07** [cmp-cc/vue-cookies#22]
We are using Vue for saving a cookie. We have the issue that cookies are not saved within Safari. 
When using Google Chrome and Firefox it is completely working. Cookies are enabled in Safari so that's not an issue there.

Is there any issue known about that?
We are using a cookie.js file with the following: 
`import Vue from 'vue';

import VueCookies from 'vue-cookies';

Vue.use(require('vue-cookies'));

VueCookies.config('23h');

export default VueCookies;`

Then import in the .Vue files with `import cookies from '../cookies.js';`
And then use it in methods with `cookies.set('bearer', response.data.access_token);`

Thanks in advance

**Comment by cmp-cc** [comment]
again test, try write it a different way, as follows

- ---
// It only needs to be configured once
// `main.js file` (program main entry file)
```
import VueCookies from 'vue-cookies';
Vue.use(VueCookies);
VueCookies.config('23h');
```

all .Vue files do not need to `import`
```
<template>
// ...
</template>
<style>
// ...
</style>
<script>
    import api from '../api/index';
    export default{
      data(){
          return{
            //
          }
      },
      created:function(){
          this.$cookies.set('bearer', response.data.access_token);

          // or
          
          // window.$cookies.set('bearer', response.data.access_token);
      },
      methods:{
        addUser: function(){
          this.$cookies.set('bearer', response.data.access_token);

          // or

          // window.$cookies.set('bearer', response.data.access_token);
        }
      },

    }
</script>
```

**Comment by lad475** [comment]
I'm also having an issue with setting cookies only on Safari- it's working perfectly on chrome and other browsers.

I'm using Vue.use(VueCookies) in my main, and then calling this.$cookies.set('token', accessToken), and while it works in other browsers, it just won't set any cookies in safari.

**Comment by cmp-cc** [comment]
fix v1.5.12
try test
